src/main/java/org/breedinginsight/brapps/importer/services/processors/GermplasmProcessor.java

```diff
@@ -242,52 +259,88 @@ public Map<String, ImportPreviewStatistics> process(List<BrAPIImport> importRows
 
             Germplasm germplasm = brapiImport.getGermplasm();
 
+            // Assign the entry number
+            if (germplasm.getEntryNo() == null) {
+                germplasm.setEntryNo(Integer.toString(i + 1));
```

Comment:
This _could_ cause a collisions.  A user defined Entry Number for a different row may have the value i+1

Reply:
I thought there was a validation to ensure that either all rows had an entry number, or no rows had an entry number?

Reply:
@davedrp Are you ok with the duplicate check Tim referenced?